File: app/src/main/java/oscar/com/eater/Response/RandomRecipeRequestResponse.java
package oscar.com.eater.Response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import oscar.com.eater.Pojo.Recipe;

/**
 * Created by omenji on 3/17/17.
 */

public class RandomRecipeRequestResponse {

    @SerializedName("recipes")
    public List<Recipe> products;

    public List<Recipe> getProducts() {
        return products;
    }

    public void setProducts(List<Recipe> products) {
        this.products = products;
    }
}

File: app/src/main/java/oscar/com/eater/Response/RandomRecipeRequestResponse.kt
package oscar.com.eater.Response

import com.google.gson.annotations.SerializedName

import oscar.com.eater.Pojo.Recipe
import oscar.com.eater.Pojo.RecipesWrapper

/**
 * Created by omenji on 3/17/17.
 */

class RandomRecipeRequestResponse {

    @SerializedName("recipes")
    var products: RecipesWrapper? = null

}
